app/presenters/concerns/hyku_addons/presenter_delegatable.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module HykuAddons
```

Comment:
I think this would be best as a class method that took an array of fields as an argument, rather than a concern with an array. That is the next step i think.